ui/pages/_app.js

```diff
@@ -138,6 +140,12 @@ class MesheryApp extends App {
     this.meshsyncEventsSubscriptionRef.current = meshSyncEventsSubscription;
   }
 
+  fullscreenchanged = () => {
```

Comment:
camelCase

Reply:
Thanks for this, @ckbhatia. 

camelCasing applied.